e_os.h

```diff
@@ -507,6 +507,7 @@ struct servent *PASCAL getservbyname(const char *, const char *);
 #    define strdup _strdup
 #   endif
 #   define unlink _unlink
+#   define fileno _fileno
```

Comment:
Which file had problem with this? Point is that this might need some re-thinking/overhaul, so I'm trying to collect information and thoughts...